scene/gui/text_edit.cpp

```diff
@@ -1891,6 +1895,39 @@ void TextEdit::_notification(int p_what) {
 				update();
 			}
 		} break;
+		case Control::NOTIFICATION_DRAG_BEGIN: {
+			selection.selecting_mode = Selection::MODE_NONE;
+			drag_action = true;
+			dragging_minimap = false;
+			dragging_selection = false;
+			can_drag_minimap = false;
+			click_select_held->stop();
+		} break;
+		case Control::NOTIFICATION_DRAG_END: {
+			if (is_drag_successful()) {
+				if (selection.drag_attempt) {
+					selection.drag_attempt = false;
+					if (!readonly && !Input::get_singleton()->is_key_pressed(KEY_CONTROL)) {
+						_remove_text(selection.from_line, selection.from_column, selection.to_line, selection.to_column);
+						cursor_set_line(selection.from_line, false);
+						cursor_set_column(selection.from_column);
+						selection.active = false;
+						selection.selecting_mode = Selection::MODE_NONE;
+						update();
+						// } else if (deselect_on_focus_loss_enabled) {
+						// 	deselect();
```

Comment:
This should be removed (it can be re-added if #54166 is background).

Reply:
It was a personal reminder, when this pull request is approved these lines must be added to https://github.com/godotengine/godot/pull/54353